eng/Npm.Workspace.nodeproj

```diff
@@ -15,9 +15,14 @@
     <PackageVersion>$(VersionPrefix)$(VersionSuffix)</PackageVersion>
   </PropertyGroup>
 
-  <Target Name="Restore">
+  <Target Name="_VerifyNPMCache">
```

Comment:
@javiercn is this the right time to call `npm cache verify` (before Restore)?

Reply:
Yep